pkgs/by-name/li/libvmi/package.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  autoconf-archive,
+  bison,
+  flex,
+  glib,
+  pkg-config,
+  json_c,
+  libvirt,
+
+  withVMIFS ? true,
+  fuse,
+
+  legacyKVM ? false,
+  libkvmi,
+
+  xenSupport ? true,
+  xen-slim,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libvmi";
+  version = "0.14.0-unstable-2024-07-25";
+  libVersion = "0.0.15";
+
+  outputs = [
+    "out" # Contains binaries helpful for testing the libraries. Must be run on the host system.
+    "lib" # Contains the LibVMI libraries. This is most likely what you want.
+    "dev" # Development headers and the pkg-config spec.
+  ];
+
+  src = fetchFromGitHub {
+    owner = "libvmi";
+    repo = "libvmi";
+    rev = "2d7258b5ab2cddea8d0889303d5f2f6f07b125b9";
+    hash = "sha256-gTNZEESHtC5K1pM9cznI3OHfq7Fg0LRrSBHybM+DTqs=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    autoconf-archive
+    bison
+    flex
+    pkg-config
+  ];
+
+  buildInputs =
+    [
+      glib
+      json_c
+      libvirt
+    ]
+    ++ (lib.lists.optional xenSupport xen-slim)
+    ++ (lib.lists.optional (!legacyKVM) libkvmi)
+    ++ (lib.lists.optional withVMIFS fuse);
+
+  configureFlags =
+    lib.lists.optional (!xenSupport) "--disable-xen"
+    ++ (lib.lists.optional legacyKVM "--enable-kvm-legacy")
+    ++ (lib.lists.optional withVMIFS "--enable-vmifs");
```

Comment:
```suggestion
    ++ lib.optional legacyKVM "--enable-kvm-legacy"
    ++ lib.optional withVMIFS "--enable-vmifs";
```

Reply:
The parenthesis removal is fine, but are you sure about removing `lists`? Declaring the library explicitly seems more reasonable to me, instead of using the generic alias. Pretty much all other PRs I've written use the explicit `lib.{lists,strings,attrsets,modules,options}.*` declaration. It may be shorter to remove it, but it's a helpful indicator to what type of data the library is handling here.